nncf/config/schema.py

```diff
@@ -729,6 +738,9 @@ def with_attributes(schema: Dict, **kwargs) -> Dict:
         # Validation of each separate compression description schema occurs in a separate step.
         # This is required for better user feedback, since holistic schema validation is uninformative
         # if there is an error in one of the compression configs.
+        "compression_lr_multiplier": with_attributes(_NUMBER,
+                                                     description="Used to increase/decrease gradients "
+                                                                 "for compression algorithms' parameters."),
```

Comment:
You duplicate the entry four times - what if we need to adjust the description? Extract the sub-dict `COMPRESSION_LR_MULTIPLIER_PROPERTY = {"compression_lr_multiplier": with_attributes(_NUMBER, ...)} ` and in each dict where the corresponding entry has to appear, do the `{..., **COMPRESSION_LR_MULTIPLIER_PROPERTY, ...}` expansion.

Reply:
Done.